frontend/components/TableContainer/DataTable/_styles.scss

```diff
@@ -357,6 +357,7 @@ $shadow-transition-width: 10px;
   &__footer {
     display: flex;
     align-items: center;
+    height: 50px; // Match pagination height as pagination is optionally rendered
```

Comment:
Fix from Janis's finding:
> Spot test that no styling looks weird/conflicts with local code
> ‼️This one looks weird:

The footer comment didn't have vertical padding if the pagination was not rendered, added padding